src/sources/util/http/prelude.rs

```diff
@@ -82,6 +85,7 @@ pub trait HttpSource: Clone + Send + Sync + 'static {
                         Ok(())
                     } else {
                         debug!(message = "Path rejected.");
+                        emit!(HttpInvalidRouteError {});
                         Err(warp::reject::custom(ErrorMessage::new(
                             StatusCode::NOT_FOUND,
                             "Not found".to_string(),
```

Comment:
Re: my comment about the `HttpInvalidRouteError` - I imagine it should better align with the messages here. On that note, this `debug` seems duplicative of the new internal event.

Reply:
Ah, good call. I've deleted the debug log and updated the Event in question.